JakeUni95/interview-questions-frontend
# Replace default header with a custom unbranded header

It removes the gov.uk logo and also changes the banner colour to red.

## How to test
1. Run CMS locally.
2. Make sure that frontend files are built by running `npm run build`.
3. Run frontend locally.
4. Click through pages and verify visual changes to header.
